src/browser/api/window.js

```diff
@@ -1207,7 +1207,7 @@ Window.moveTo = function(identity, x, y) {
     const safeX = toSafeInt(x);
     const safeY = toSafeInt(y);
 
-    browserWindow.setBounds({
+    browserWindow.setBounds({ // no need to call clipBounds as width and height not changing
```

Comment:
Sounds like an extra/unnecessary comment to me.
But if we keep it, probably should go onto a separate line.